R/uvhydrograph-render.R

```diff
@@ -180,47 +180,44 @@ createPrimaryPlot <- function(
 
   #Don't add the right-side axis if we aren't actually plotting anything onto it
   if((limsAndSides[['sides']][['reference']] == 4) || (limsAndSides[['sides']][['comparison']] == 4)){
-    plot_object <- lines(plot_object, x=0, y=0, side = 4, reverse = primarySeriesList[['inverted']]) %>%
+     plot_object <- lines(plot_object, x=0, y=0, side = 4, reverse = primarySeriesList[['inverted']]) %>%
         axis(side = 4, las = 0, reverse = primarySeriesList[['inverted']])
   }
   
+  #uncorrected data
+  if(!isEmptyOrBlank(primarySeriesList[['uncorrected']]) && !isEmptyVar(primarySeriesList[['uncorrected']][['points']])) {
+    plot_object <- plotTimeSeries(plot_object, primarySeriesList[['uncorrected']], "uncorrected", 
+                                  timezone, getPrimaryPlotConfig, list(uvInfo[['label']], limsAndSides$ylims[['primary']], limsAndSides$sides[['primary']]), excludeZeroNegativeFlag)
```

Comment:
The `doLog` that's used in `getPrimaryPlotConfig` is calculated within `plotTimeSeries`. 

When `logPrimary` is calculated (line 200) that is used for the section further down that limits other items being added on that axis to only having positive values (line 222)